### DISCUSSION TITLE
有没有办法主动添加别人

### DISCUSSION THREAD

**Discussion by GSCSDSUN** [mamoe/mirai#1041] [Q&A]
正在使用Javafx+Mirai编写QQ的第三方客户端 但是好像不能主动添加别人。。。

**Comment by Ultronxr** [comment]
README[不会支持的协议](https://github.com/mamoe/mirai#%E4%B8%8D%E4%BC%9A%E6%94%AF%E6%8C%81%E7%9A%84%E5%8D%8F%E8%AE%AE)

不读文档的结果

**Reply by GSCSDSUN** [reply]
readme读了

**Reply by GSCSDSUN** [reply]
在想办法支持

**Reply by sandtechnology** [reply]
切勿玩火，___________

**Reply by GSCSDSUN** [reply]
因为linux qq真的太垃圾了 而且我是armhf架构 根本用不了QQ
